### PR TITLE
Doc/code review guidelines

### PR DESCRIPTION
**Pull request by Croydon-Brixton**
Add documentation for code review and update CI and Pre commits

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -3,35 +3,33 @@
...
-    - name: Lint with flake8
-      run: |
-        # stop the build if there are Python syntax errors or undefined names
-        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-        # lint with the rules defined in setup.cfg
-        make lint
-      
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
```
Comment (Croydon-Brixton) [inline .github/workflows/ci.yml:19]: I don't understtand this line. Can you explain this in more detail>

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+repos:
```
Comment (Croydon-Brixton) [inline .pre-commit-config.yaml:1]: Looks good to me (LGTM)

**Review by Croydon-Brixton — COMMENTED** [review]
Commented stuff for demo

**Review by rdnfn — APPROVED** [review]
Looks great!

**Pull request merged**